### ISSUE TITLE
showstopper glitch when duplicating a project

### ISSUE THREAD

**Issue by MJWainwright** [epistemonikos/isoq#214]
@damian-garrido . We were not able to use iSoQ during the in-person training at the Cochrane Colloquium in London last week because of a major glitch in iSoQ. The following occured:

- Simon requested that I move two columns from the characteristics of studies table to the methodological limitations table in our workshop template FASD (I had used this template in June during a workshop without a problem. The only change was the action of deleting and adding columns before duplication). 
- I added the two columns and data manually to the methodological limitations table and then deleted the two columns from the characteristics of studies tables. 
- I then duplicated the project and shared the duplicate with Simon.
- the duplicate was unchanged - that is, it did not reflect the changes I had made to the methodological limitations and characteristics of studies table. 
- After realising we could not fix the bug, I shared the template with him (without duplicating), but then when he had to duplicate the project (one for each small group attending the workshop), the same and other errors occured. 
- Really weird things are happening! I will share the following project and its copy with you "TEMPLATE (facilitator version) - GRADE-CERQual Workshop : FASD Example". When you compare them you will see:
          1. in the Copy the tables have the wrong information is them. It's information from another iSoQ project
          2. instead of having 14 references (as in the original), the copy has 486!
          3. instead of 1 review finding on the iSoQ table, there's a whole bunch from different projects
It is as though iSoQ has merged the data for this project with other projects. 
I wondered whether, regardless of the origin of the bug whether we should have a similar approach to duplicate project as we do to publish project - that is, that the user cannot click out or cancel the process until the save is finished, and that the close button does not appear until the save process is finished. 

In the original template:
![image](https://github.com/epistemonikos/isoq/assets/72507850/119d3378-9106-4347-bf4c-8e26ee97e416)

In the copy:
![image](https://github.com/epistemonikos/isoq/assets/72507850/287b2976-549f-4ec9-9a4d-65daef9932f5)

In the original:
![image](https://github.com/epistemonikos/isoq/assets/72507850/168f3d5a-d65d-43e3-ba93-4b6023f90e76)

In the copy:
![image](https://github.com/epistemonikos/isoq/assets/72507850/72f68871-940b-45ac-bc52-4220a3f90823)

In the original:
![image](https://github.com/epistemonikos/isoq/assets/72507850/8391a8e4-f24f-4f7b-a060-40cb1b3b9321)

In the copy:
![image](https://github.com/epistemonikos/isoq/assets/72507850/0afc1851-62ee-4fa4-b963-836c17a1dcf9)

In the original:
![image](https://github.com/epistemonikos/isoq/assets/72507850/bdcca2b1-c5d1-4e9f-9e5a-0ebe35c48d60)

In the copy:
![image](https://github.com/epistemonikos/isoq/assets/72507850/7c8f195f-3027-42b6-ad1a-5848e858e4c7)

**Comment by damian-garrido** [comment]
@MJWainwright could you check if this issue was resolved on the test server please

**Comment by MJWainwright** [comment]
@damian-garrido . I will test this early October. Thanks!

**Comment by MJWainwright** [comment]
@damian-garrido I tested this on the test server following the same steps (adding two columns and content in one table, deleting two columns in the other and then duplicating the project). I did not see an error. However, I do not have the same project in the test server as I do in the production server account...not sure if this could influence things.

I think go ahead and publish to the production server, let me know when that's done and I will test it on the production server with the project that caused the problems. thanks!
